samples/samples.json

```diff
@@ -697,6 +697,17 @@
                     "Audio"
                 ]
             },
+            {
+                "title": "Custom Capabilities Filter - MediaCapabilitiesAPI",
+                "description": "This sample shows how to filter representations based on responses from MCA.",
```

Comment:
Let's use the full name here instead of the abbreviation : "Media Capabilities API"